_data/menu_docs_dev.json

```diff
@@ -360,7 +360,7 @@
               ]
             },
             {
-              "page": "Appender (C++/Java)",
+              "page": "Appender (C++/Go/Java)",
```

Comment:
Worth including a link to https://duckdb.org/docs/archive/0.9.2/api/c/appender ? And the rust one too?

Reply:
Sure, buth then I'll drop the "language X/language Y/..." thing